### ISSUE TITLE
Should be able to get credentials stored in Jenkins

### ISSUE THREAD

**Issue by efrangul** [jenkinsci/urltrigger-plugin#44]
### Feature Request

Instead of having only user/password in Basic Authentication it would be good to get credentials stored in Jenkins Credentials.

**Comment by flabrie** [comment]
I strongly agree with this. We need to update credentials on a regular basis, and having to recall we need to do it in dozens of Jenkins build job is really painful.
